### ISSUE TITLE
Mech balancing.

### ISSUE THREAD

**Issue by stubbed** [Anuken/Mindustry-Suggestions#63]
Before I start my proposal changes, I would like to clarify that I am not a game developer. I have no idea how balanced my suggestions will turn out to be, or how much exactly they will affect the game. I am just throwing some ideas I have had in mind for some time.

**Describe what you would like changed, and why.**
The new mechanics I am about to suggest my seem odd and unfitting. I will try my best to give a clear explanation for everything.

- Alpha mech:No change.

Reason for changes: Similar to the default ship, the alpha mech has nothing special besides being a ground unit. It has more firepower and can avoid air only defenses, but it comes at the cost of moving slower.

-Delta mech:

1. Energy bar, limits the amount of attacks.

2. Generates energy by moving, the energy generated is used to fire.

3. Each shot uses 3-4% energy.

4. Special: Precise long beam (Similar to the Lancer's beam) that uses 10% energy/s.

![Delta_special](https://user-images.githubusercontent.com/28898462/77623298-e8155380-6f50-11ea-8cd7-012b97923628.jpg)

Reason for changes: The Delta mech is possibly the most underwhelming mech when it comes to value. Its attacks are slow and inaccurate, and it being a mech makes it have slower movement. And therefore, I believe that having a special attack is not something questionable for a mech that is this underwhelming. Of course with something great as a rapid Lancer beam, a line has to be drawn somewhere to keep balance, so I propose the addition of an "overheat" mechanic for firing, which in this case is the mech's energy bar.

- Tau mech:

1. Energy bar, limits attacks and healing.

2. Uses solar panels to generate energy passively (and at a lower rate in dark environments), and can also generate energy by moving.

3. Each shot and healing aura uses 5% of the energy.

4. Ability to carry 70 liquid units of cryofluid and water. Cryofluid will reduce the amount of energy used by half.

5. Water aura: When walking near fires, the Tau mech will use the water carried to extinguish the fires. This is the only use for carrying water.

6. Special: Heals units and buildings around it by 10%/s, uses 20% energy/s.

 
![tau_special](https://user-images.githubusercontent.com/28898462/77625977-a1762800-6f55-11ea-8dfc-296cb8dac1ee.jpg)

Reason for changes: Since the Tau mech is the only way you are able to manually heal, it is apparent to me that this mech should be focused on support. The ability to extinguish fires would be insanely useful, as burning oil fields are currently a very big hassle to extinguish. With the addition of a "special", the Tau mech may seem like it would be overpowered. My reason for making the Tau "overpowered" if you will call it, is to create diversity. As of right now, the Omega mech is considered the best because of its damage potential, why choose a mech that is mediocre at best when you can have very high damage output?

- Omega mech: 

1. Weapons overheat by 10% everytime you fire.

2. If weapons overheat by 100%, the mech will catch fire and weapons will cool down after 8s.

3. Able to carry cryofluid, reduces the overheating by half.

4. Special: shoots one big bomb that goes in a straight line and has the damage of all the small shots combined. Overheats weapons by 50%.

![omega_special](https://user-images.githubusercontent.com/28898462/77629227-3d099780-6f5a-11ea-8db0-9a2b56040273.jpg)

Reason for changes: As stated before, the Omega is easily the best mech. Offering high damage output and high health, there is no other mech that does its things better than this  does its own. However, I do not feel like the Omega should get a big nerf, even though I stated before I am trying to create diversity. The Omega mech should be a stronger mech than others because of the high cost to obtain it.

**Describe the changes you want to propose. Include possible alternatives.**
In summary: The mechs should be more balanced towards creating diversity. As it stands right now, mechs feel like upgrades that you will go through to eventually get the best one.

Alternatively, since mechs feel like an upgrades at the moment, it would make sense if more mechs were added and sorted to tiers, highest tier being the strongest and most expensive.

**Comment by DS-3K** [comment]
Having the Tau Mech be able to heal units better would be essential for PVP maps in wich player damage is set to 0
